docs/pages/components/popover.md

```diff
@@ -24,6 +24,7 @@ The basic wrapping structure of a popover.
 * `fd-popover` - wrapper for the whole container
 * `fd-popover__control` - control element to toggle the display of the popover
 * `fd-popover__body` - wrapper that contains the popover content for CSS-only implementations
+* `fd-popover-body__wrapper` - wrapper for the body to prevent overflow
```

Comment:
It wouldn't make sense to re add an example because they are only to be used with things like combobox so it would confuse user to have combobox in popover.